### PR TITLE
Toggle Bug Fixed

### PR DESCRIPTION
**Pull request by devansh-webkul**

### CODE REVIEW

File: docs/packages/README.md [hunk 1]
```diff
@@ -1,30 +1,63 @@
 # Package Development
 
-A package is a unit added to your application for enhancement which includes routes, controllers, views, and configuration specifically. Packages are created to manage your large applications into smaller units.  
-In the bagisto, we have created plenty of packages at path `packages/Webkul/`. You can find a basic tree-structure of package below:
+A package is a unit added to your application for enhancement which includes routes, controllers, views, and configuration specifically. Packages are created to manage your large applications into smaller units.
 
-```php
-- module-name/
+In the Bagisto, we have created plenty of packages at path `packages/Webkul/`. You can find a basic tree-structure of the package below:
```
Comment (ghermans) [inline docs/packages/README.md:5]: - In the Bagisto, we have created plenty of packages at path `packages/Webkul/`. You can find a basic tree-structure of the package below:

Should be: 

In Bagisto, we have created plenty of packages inside `packages/Webkul/`. You can find a basic tree-structure of the package below:

File: docs/packages/add_menu_in_admin.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      public function register()
+      {
+          $this->mergeConfigFrom(
+              dirname(__DIR__) . '/Config/admin-menu.php', 'menu.admin'
+          );
+      }
+  }
+  ~~~
+
+- Now, menu will display in the admin panel. You can change the icon according to your needs.
```
Comment (ghermans) [inline docs/packages/add_menu_in_admin.md:66]: Now, menu will display in the admin panel. You can change the icon according to your needs.

Should be 

Now, the menu will display in the admin panel. You can change the icon according to your needs.

**Review by ghermans — CHANGES_REQUESTED** [review]

File: docs/packages/README.md (hunk 1 above)
Comment (devansh-webkul) [reply docs/packages/README.md]: Done!

File: docs/packages/add_menu_in_admin.md (hunk 2 above)
Comment (devansh-webkul) [reply docs/packages/add_menu_in_admin.md]: Done

**Pull request merged**